src/quo2/foundations/colors.cljs

```diff
@@ -91,6 +91,11 @@
 (def white-opa-90 (alpha white 0.9))
 (def white-opa-95 (alpha white 0.95))
 
+;;;;Black
+
+;;Solid
+(def black "#000")
```

Comment:
hmm, did you double check with the design team that we want this and it's not just something that was left over when they adjusted the colors?

Reply:
I'll double check with the design team :)

Reply:
Actually I double checked designs and we are good using `neutral-100`. Just updated PR with this change